src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.model;
+
+import java.util.Objects;
+
+public final class Fruit {
+    private final String fruit;
```

Comment:
```suggestion
    private final String name;
```
add changes to constructor and all methods of this class accordingly